pkg/apis/kf/v1alpha1/space_types.go

```diff
@@ -95,10 +97,30 @@ type SpaceSpecExecution struct {
 	Env []corev1.EnvVar `json:"env,omitempty" patchStrategy:"merge" patchMergeKey:"name"`
 }
 
+// SpaceSpecResourceLimits contains definitions for resource usage limits.
+type SpaceSpecResourceLimits struct {
+	// SpaceQuota holds the k8s ResourceQuota created for the whole space.
+	// For now, only one ResourceQuota per space is supported.
+	// Consider allowing multiple ResourceQuotas when more quota scopes are enabled in k8s
+	// +optional
+	// +patchMergeKey=name
+	// +patchStrategy=merge
+	SpaceQuota corev1.ResourceList
```

Comment:
This should have a JSON tag for serializing/deserializing

Reply:
done